TELCODOCS-368: Deploying an OCP cluster manually without Assisted Installer (regardless of use case) (#38068)
openshift/openshift-docs: modules/install-sno-requirements-for-installing-on-a-single-node.adoc

Installing {product-title} on a single node alleviates some of the requirements of high availability and large scale clusters. However, you must address the following requirements:

> * *Administration node:* You must have an administration node or laptop to prepare the ISO, to create the USB boot drive, and to monitor the installation.

Reviewer: I'm not sure node is the correct term here either. Does "node" connotate a host that is installed in the cluster? Would host be better? 

```suggestion
* *Administration host:* You must have a computer to prepare the ISO, to create the USB boot drive, and to monitor the installation.
```

Author: Switched to host.
